### ISSUE TITLE
[YCQL] select count(*) can paginate

### ISSUE THREAD

**Issue by mbautin** [yugabyte/yugabyte-db#10112]
Jira Link: [DB-4643](https://yugabyte.atlassian.net/browse/DB-4643)
Sometimes SELECT COUNT(*) FROM table_name would produce one row and then print --more-- and produce more results.

With "paging off", this does not happen, and only one result row is produced.

I think pagination in count(*) is unexpected, at least from a point of view of SQL.

Use --request-timeout=3600 or something similar for such tables.

**Comment by nocaway** [comment]
Proxy server runs paging on one tablet at a time and causes the issue.  To reproduce,
- Create a cluster with many tablet servers such as `--replication_factor 7`
- Start `ycqlsh --request-timeout=3600`
- Run the following statements.

```
ycqlsh> use yuga;
ycqlsh:yuga> create table tab(i int primary key, j int, k int);
ycqlsh:yuga> insert into tab(i, j, k) values(1, 1, 1);
ycqlsh:yuga> insert into tab(i, j, k) values(2, 2, 2);
ycqlsh:yuga> insert into tab(i, j, k) values(3, 3, 3);
ycqlsh:yuga> insert into tab(i, j, k) values(4, 4, 4);
ycqlsh:yuga> insert into tab(i, j, k) values(5, 5, 5);
ycqlsh:yuga> paging 1;
Page size: 1
ycqlsh:yuga> select count(*) from tab;

 count
-------
     2

---MORE---
 count
-------
     1

---MORE---
 count
-------
     1

---MORE---
 count
-------
     1

(4 rows)
```

**Comment by nocaway** [comment]
Workaround -  Turn paging OFF.
1. On `ycqlsh`, use PAGING command.
- Command `PAGING ON` will set page size to 100.
- Command `PAGING OFF` will set page size to max integer (9223372036854775807)
- Command `PAGING 777` will set page size to `777`
```
ycqlsh:test> PAGING OFF;
Disabled Query paging.
ycqlsh:test> PAGING ON;
Now Query paging is enabled
Page size: 100
ycqlsh:test> PAGING 777;
Page size: 777
```

2. In a java program, use function `setFetchSize()`
```
    SimpleStatement stmt = new SimpleStatement("SELECT * FROM test_in_paging ");
    stmt.setFetchSize(Integer.MAX_VALUE);
```
- When the above statement is executed, its page size is set to max integer. This is the same as turning `PAGING OFF` on ycqlsh
- The default page size in JAVA program is 5000
- When invalid values (e.g. negative integer or 0) are passed to `setFetchSize`, default value of 5000 will be used to execute.

**Comment by m-iancu** [comment]
Closing as duplicate of https://github.com/yugabyte/yugabyte-db/issues/13295 (fixed in latest, pending 2.16 backport).
